lib/internal/url.js

```diff
@@ -367,7 +367,7 @@ class URLSearchParams {
             if (pair.length !== 2) {
               throw new ERR_INVALID_TUPLE('Each query pair', '[name, value]');
             }
-            // Append (innerSequence[0], innerSequence[1]) to querys list.
+            // Append (innerSequence[0], innerSequence[1]) to query's list.
```

Comment:
```suggestion
            // Append (innerSequence[0], innerSequence[1]) to the query's list.
```
I didn't read over the context, so take this with a grain of salt, but I think the grammar needs `the`

Reply:
The comment is the verbatim spec text from https://url.spec.whatwg.org/#interface-urlsearchparams